std/array.d

```diff
@@ -1298,26 +1298,45 @@ Alias for $(XREF algorithm, splitter).
 deprecated("Please use std.algorithm.splitter instead.") alias splitter = std.algorithm.splitter;
 
 /++
-Eagerly splits $(D s) into an array, using $(D delim) as the delimiter.
+    Eagerly splits $(D r) into an array, using $(D sep) as the delimiter.
 
-See_Also: $(XREF algorithm, splitter) for the lazy version of this operator.
+    The range must be a $(LINK2 std_range.html#isForwardRange, forward range).
```

Comment:
I've replace LINK2 with FULL_XREF. I have no idea where this stuff is documented, though. I don't see FULL_XREF in doc.dd, though LINK2 is there.